blog/2024/09/juliacon-2025-announcement.md

```diff
@@ -0,0 +1,102 @@
+
+## JuliaCon Global 2024 Wrap Up
+
+JuliaCon 2024 was a wonderful, record breaking event for our community. If you were able to attend in person we hope you enjoyed the conference and plan to attend future JuliaCons! If you were an online viewer or have been watching our talks as they've been uploaded on YouTube hopefully you plan to join us in 2025!
+
+![JuliaCon Crowd](/assets/blog/2024-JuliaCon/140A0016.jpg)
+
+### Julia Community Prizes
+
+We want to be sure you didn’t miss the Julia Community Prizes. These prizes were announced at the closing session of JuliaCon 2024 on July 12th, but if you missed it, the awardees this year were:
+
+\- **The JuMP Team**: Including **Oscar Dowson**, **Benoît Legat**, and **Miles Lubin**, for their long-term commitment to JuMP.jl and the broader mathematical programming ecosystem in Julia.
+
+\- **The Makie Team:** Including **Simon Danisch**, **Julius Krumbiegel**, **Frederic Freyer**, and **Anshul Singhvi**, for their creation of Makie.jl and improvements to the state of data visualization in Julia.
+
+\- **Guillaume Dalle:** For his various contributions to the Julia ecosystem including DifferentiationInterface.jl and ModernJuliaWorkflows.
+
+Awardees are nominated by the Julia community, and then winners are selected by a committee of senior community members based on their holistic contributions across the ecosystem.
+
+### JuliaCon 2024 Organizers and Volunteers
+
+JuliaCon wouldn’t happen without an amazing group of organizers and volunteers. The organizing committee put in an incredible amount of work over the past year, and an incredible 70+ volunteers made the conference possible.
+
+I first want to highlight the incredible Local Organizing Committee. Led by Gareth Thomas, who is unmatched in his ability to bring people together for a common cause, the local committee was the key to both JuliaCon Local Eindhoven and JuliaCon Global 2024!
+
+\- Gareth Thomas, Version Bay: Local Chair
+
+\- Jorge Vieyra, ASML: Local Organizer
+
+\- Max van Dijk, ASML: Local Organizer
+
+\- Erwin Hoogerwoord, TU/e: Local Organizer + Workshops
+
+\- Dmitry Bagaev, TU/e: Local Organizer + Web
+
+\- Long Chen, ASML: Local Organizer
+
+This local committee is a fantastic blueprint for future JuliaCon Locals, and for local committees in the future.
+
+Our global committee this year was also the biggest we’ve had in years, and organized our wonderful program, diversity initiatives, and sponsorship roster. 
+
+\- Katie Hyatt, AWS: Program co-Chair
+
+\- Luca Ferranti, PlantingSpace: Program co-Chair
+
+\- Mark Kittisopikul, HHMI: Finance Chair
+
+\- Skylar Gering, CalTech: Diversity & Accessibility co-Chair
+
+\- Valentin Churavy, MIT: Diversity & Accessibility co-Chair
+
+\- Carsten Bauer: Proceedings Chair
+
+\- Avik Sengupta, JuliaHub: Organizer
+
+\- Ranjan Anantharaman, JuliaHub: Executive co-Chair
+
+\– Raye Kimmerer, MIT: Executive co-Chair
```

Comment:
Any reason for not using proper bullet lists throughout the post?

Reply:
Also, if anything I presume this should be
```suggestion
\- Raye Kimmerer, MIT: Executive co-Chair
```

The difference is
```julia-repl
julia> '–'
'–': Unicode U+2013 (category Pd: Punctuation, dash)

julia> '-'
'-': ASCII/Unicode U+002D (category Pd: Punctuation, dash)
```

Reply:
Haha I think this is some weird artifact of copying from my Markdown editor into a different flavor? No idea why this would happen. Thanks for noticing

Reply:
I copied directly from Typora... Maybe I need to force it to use a different markdown flavor